### DISCUSSION TITLE
Slint and wayland

### DISCUSSION THREAD

**Discussion by schnapper79** [slint-ui/slint#2758] [Q&A] [answered]
Maybe a stupid question but I am all new to wayland, using X11 before... I am running an old imx6 system... Do I need a separate compositor to run a full screen slint ui with wayland?

**Comment by tronical** [comment, marked as answer]
Yes, at the moment that is necessary. That compositor will just pass through the buffers produced by the application - so no extra time is spent on composition. We don't have an "eglfs" equivalent yet. I've been playing with this using Vulkan - which offers an elegant API for that that works cross-platform - but that "old imx6" system might not have it :)

See also #712 for the corresponding issue.

**Reply by flukejones** [reply]
I'm fairly certain that those + similar ARM systems do not have vulkan support (via mesa and etnaviv driver) but I haven't actually looked

**Comment by schnapper79** [comment]
Imx6 does not support Vulkan, the newer imx8 does (afaik)

Luke Jones ***@***.***> schrieb am Di., 23. Mai 2023, 03:42:
